I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 3, in script
    init python:
  File "game/script.rpy", line 4, in <module>
    from game.lib.zhongwen.number import 中文數字
TypeError: unsupported operand type(s) for |: 'type' and 'type'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 3, in script
    init python:
  File "D:\renpy\renpy-8.3.0-sdk\renpy\ast.py", line 821, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\renpy\renpy-8.3.0-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 4, in <module>
    from game.lib.zhongwen.number import 中文數字
  File "D:\farming-game\Farming Game\game\lib\zhongwen\number.py", line 15, in <module>
    def 轉數值(n, 傳回格式=False) -> int | float:
TypeError: unsupported operand type(s) for |: 'type' and 'type'

Windows-10-10.0.22000 AMD64
Ren'Py 8.3.0.24082114
鬼谷三家村 1.0.0
Sat Aug 31 01:01:53 2024
